### PR TITLE
Upper Case

### PR DESCRIPTION
**Pull request by Ran73Cohen**
UPPER CASE BAD!
Camel Case Good :D

### CODE REVIEW

**Comment by Meytal2021** [comment]
@meytal2021

**Comment by ashachaf** [comment]
looks good to me.

File: src/components/layout/SettingsMenu.vue [hunk 1]
```diff
@@ -80,7 +80,7 @@
     <b-dropdown-divider v-if="!showLocale"></b-dropdown-divider>
     <b-dropdown-group id="dropdown-group-3" v-if="!showLocale">
       <b-dropdown-header
-        class="text-uppercase"
+        class=""
```
Comment (pingustar) [inline src/components/layout/SettingsMenu.vue:83]: can be removed

File: src/components/modals/ModalVoteDetails.vue [hunk 2]
```diff
@@ -72,7 +72,7 @@
       </template>
 
       <template #cell(voted)="data">
-        <div class="text-uppercase" :class="getVoteClass(data.value)">
+        <div class="" :class="getVoteClass(data.value)">
```
Comment (pingustar) [inline src/components/modals/ModalVoteDetails.vue:75]: ^

File: src/components/vote/proposals/OpenProposals.vue [hunk 3]
```diff
@@ -181,7 +181,7 @@
             >
               <div class="row">
                 <span class="col-3">
-                  <span class="text-uppercase">{{ item.votes.voted }}</span>
+                  <span class="">{{ item.votes.voted }}</span>
```
Comment (pingustar) [inline src/components/vote/proposals/OpenProposals.vue:184]: ^

**Review by pingustar — CHANGES_REQUESTED** [review]
missing cleanup of empty class attributes

**Comment by Ran73Cohen** [comment]
> 
> 
> missing cleanup of empty class attributes

Yeah I noticed it and forgot to fix :|

**Review by pingustar — APPROVED** [review]

**Comment by ashachaf** [comment]
what is holding this off from being pushed to prod?

**Comment by Ran73Cohen** [comment]
@Meytal2021 Can you review the new font sizes?

**Pull request merged**